src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -817,16 +819,21 @@ private function getCategoryToolbarButtons(Request $request)
             ];
         }
 
-        $categoryId = $request->attributes->get('categoryId');
+        if (empty($categoryId)) {
```

Comment:
Nice to have.
Here empty is not necessary, you can just use this suggestion. Because of your type-hint in int.
```suggestion
        if ($categoryId === 0) {
```

Reply:
Fixed

Reply:
Thank you